website/docs/r/branch_policy_build_validation.html.markdown

```diff
@@ -62,6 +62,12 @@ resource "azuredevops_branch_policy_build_validation" "example" {
       repository_ref = "refs/heads/releases"
       match_type     = "Prefix"
     }
+    
+    scope {
+      repository_id  = azuredevops_git_repository.example.id
+      match_type     = "DefaultBranch"
+      repository_ref = null
```

Comment:
`repository_id` and `repository_ref` should not set when  `match_type     = "DefaultBranch"`
```suggestion
      match_type     = "DefaultBranch"
```